_assignments/grad_assignment.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: assignment
+title: "4 Credit Hour Assignment"
+parts:
+  - title: Proposal
+    due: Mar. 25, 2019 @ 11:59 PM
+  - title: Results
+    due: Apr. 22, 2019 @ 11:59 PM
+  - title: Final Submission
+    due: day of Final Project
```

Comment:
Do we want it due the day of the final project presentation or earlier? That way we have some time to grade during finals week.

Reply:
I'll change it to the last day of class, that way they don't have to worry about it while studying for finals as well.